src/remote/agency.service.interface.ts

```diff
@@ -3,4 +3,6 @@
  */
 export abstract class IAgencyService {
     abstract spinUpAgent(walletId: string, walletKey: string, adminApiKey: string, seed: string, agentId: string): Promise<any>;
```

Comment:
Are we keeping this around "just in case"? Or is there a use for it that multi-tenant doesn't address?

Reply:
Ya, there's the possiblity that someone wants to have their agent spun up completely separately from the multitenant process - we don't have a real use case yet, but wanted to keep support for it around.

Reply:
I see. It makes the code a little bit confusing to work with, unfortunately. Especially since the verbs are different, it's unclear that they do the same thing using a different method... could you add a comment to `spinUpAgent()` explaining the difference between the two and why someone might use one over the other? Otherwise, it just looks like dead code.

Reply:
Sorry, I missed that you had already added the comments!